wifi-subsys/components/network/CMakeLists.txt

```diff
@@ -1,2 +1,4 @@
 file(GLOB_RECURSE SOURCES src/*.c)
-idf_component_register(SRCS ${SOURCES} INCLUDE_DIRS src ../../main REQUIRES storage)
+
+    idf_component_register(SRCS ${SOURCES}
+    INCLUDE_DIRS src ../../main REQUIRES storage esp-tls esp_http_client)
```

Comment:
New line at EOF

Reply:
Rocio this still mark EOF error solve it please.